manifests/crdvalidation.yaml

```diff
@@ -0,0 +1,28 @@
+# limit bundle name length up to 40
+- op: add
```

Comment:
Very cool -- this is exactly what I was looking to implement -- looks great.

Reply:
As a follow-up, I think it'd be nice to organize all the manifests a bit better (I'm partial to the way kubebuilder/operator-sdk has `config/<component>` directories and then a `config/default` that ties everything together.

Reply:
OK.  I'll open a follow up issue for it.

Reply:
https://github.com/operator-framework/rukpak/issues/228